doc/source/debugging.rst

```diff
@@ -101,6 +105,10 @@ additional options---and scoped to the ``'bluesky'`` logger with bluesky's
 :class:`bluesky.log.LogFormatter`. It streamlines common use cases without
 interfering with more sophisticated use cases.
 
+We recommend that facilities using bluesky leave this functions for users and
```

Comment:
We recommend that facilities using bluesky leave this "function"? for users and
"configure"? any standardized...